Resources/Prototypes/XenoArch/triggers.yml

```
@@ -40,6 +40,8 @@
 - type: xenoArchTrigger
   id: TriggerHeat
   tip: xenoarch-trigger-tip-heat
+  triggerBlacklist: # DeltaV
```

Change the dV comments here to explain what was added
```suggestion
  triggerBlacklist: # DeltaV - add blacklist of conflicting triggers
```

---

Isn't "add blacklist of conflicting triggers" already obvious when you read the property name "triggerBlacklist" that this comment is on? (and then also see the context of the line below it)

I don't mean to be argumentative, but I see blank `# DeltaV` comments *everywhere* in places where the entire pointed-at feature has been added (in VSCode, search `*.yml` files for regex `# DeltaV(?!.)`  ). I take review requests like this seriously because I try to integrate them into my habits / update my coding style to minimize review issues in future PRs.

---

I'm a huge proponent of self-documenting code, too, so my main point here is the property name is already "triggerBlacklist". If there's something unclear about that, do you have a suggestion on how I could change the property name?

---

> Isn't "add blacklist of conflicting triggers" already obvious when you read the property name "triggerBlacklist" that this comment is on? (and then also see the context of the line below it)

Without clarification it's unclear if it's just added to this item, newly implemented, or changed from parent, unless you look further